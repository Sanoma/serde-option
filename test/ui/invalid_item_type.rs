use serde_option::serde_option;

#[serde_option]
const FOO: usize = 42;

fn main() {}
