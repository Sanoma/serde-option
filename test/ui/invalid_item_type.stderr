error: The attribute can only be applied to struct or enum definitions.
 --> test/ui/invalid_item_type.rs:3:1
  |
3 | #[serde_option]
  | ^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `serde_option` (in Nightly builds, run with -Z macro-backtrace for more info)
